src/app/courses/pages/new-course-page/new-course-page.component.html

```diff
@@ -1 +1,66 @@
-<p>new-cours-page works!</p>
+<div class="form-wrapper">
+    <form class="new-course__form" [formGroup]="formNewCourse" (submit)="onSubmit()">
+        <h3>New course</h3>
+        <div class="new-course__input-wrapper">
+          <label for="new-course__input">
+            <span>Title:</span>
+          </label>
+          <input
+            formControlName="title"
+            class="new-course__input"
+            name="title"
+            type="text"
+            placeholder="Text input"
+          />
+        </div>
+        <div class="new-course__input-wrapper">
+          <label for="new-course__input">
+            <span>Discription:</span>
+          </label>
+          <textarea
+            cols="30"
+            rows="5"
+            class="textarea"
+            placeholder="Add description"
+            name="description"
+            required
+          ></textarea>
+        </div>
+        <div class="new-course__input-wrapper">
+          <label for="new-course__input">
+            <span>Duration:</span>
+          </label>
+          <input
+          placeholder="minutes"
+            formControlName="duration"
+            class="new-course__input duration"
+            type="text"
+          />
+        </div>
+        <div class="new-course__input-wrapper">
+          <label for="new-course__input">
+            <span>Date:</span>
+          </label>
+          <input
+            formControlName="date"
+            class="new-course__input duration"
+            type="date"
+          />
+        </div>
+        <div class="new-course__input-wrapper">
+          <label for="new-course__input">
+            <span>Authors:</span>
+          </label>
+          <input
+          placeholder="Start typing"
+            formControlName="authors"
+            class="new-course__input authors"
```

Comment:
If you use BEM for structuring your CSS, you should use a modifier here. Something like `.new-course__input new-course__input--authors`.

Anyway, there should be a separate component, so it is not necessary here at all (in other places too).